cns/ipampool/monitor.go

```diff
@@ -101,6 +103,15 @@ func (pm *Monitor) Start(ctx context.Context) error {
 			subnetCIDR = nnc.Status.NetworkContainers[0].SubnetAddressSpace
 			subnetARMID = GenerateARMID(&nnc.Status.NetworkContainers[0])
 
+			pm.metastate.primaryIPAddresses = make(map[string]struct{})
+			// Add Primary IP to Map, if not present.
+			// This is only for Swift i.e. if NC Type is vnet.
+			for i := 0; i < len(nnc.Status.NetworkContainers); i++ {
+				if nnc.Status.NetworkContainers[i].Type == v1alpha.VNET {
```

Comment:
I'd suggest checking if this is `v1alpha.VNET` or `""`, since it is possible that this CNS could be operating on NNCs created by a DNC-RC that does not yet set the type, in which case they are of type VNET